### PR TITLE
Close wallet refactor

### PR DESCRIPTION
**Pull request by banklesss**

### CODE REVIEW

File: src/hooks/index.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {useNetInfo} from '@react-native-community/netinfo'
-import {useFocusEffect} from '@react-navigation/native'
+import {useFocusEffect, useNavigation} from '@react-navigation/native'
```
Comment (wolverineks) [inline src/hooks/index.ts:3]: this file shouldn't have any react native stuff in it, out shouldn't have anything yoroi-mobile specific in it

File: src/hooks/index.test.ts [hunk 2]
```diff
@@ -5,6 +5,20 @@ import {mockWallet} from '../../storybook/mocks/wallet'
 import {TxStatusRequest, TxStatusResponse} from '../yoroi-wallets/types/other'
 import {fetchTxStatus} from '.'
 
+// Hack!! The following native modules are breaking the test
```
Comment (wolverineks) [inline src/hooks/index.test.ts:8]: which test?

File: src/hooks/index.ts [hunk 3]
```diff
@@ -92,6 +100,62 @@ export const useEasyConfirmationEnabled = (wallet: YoroiWallet) => {
...
+    ...options,
+    mutationFn: async (walletMeta) => {
+      const [newWallet, newWalletMeta] = await walletManager.openWallet(walletMeta)
+      return {
+        wallet: newWallet,
+        walletMeta: newWalletMeta,
+      }
+    },
+    onSuccess: ({wallet, walletMeta}, variables, context) => {
+      selectWalletMeta(walletMeta)
```
Comment (wolverineks) [inline src/hooks/index.ts:131]: pass aaaaallllll of this stuff in as parameters

**Review by wolverineks — CHANGES_REQUESTED** [review]

**Comment by banklesss** [comment]
Closing. A simpler fix will be applied.

**Pull request closed without merging**